webserver/observe/tracing/src/main/java/io/helidon/webserver/observe/tracing/TracingObserverConfigBlueprint.java

```diff
@@ -68,16 +70,28 @@ interface TracingObserverConfigBlueprint extends ObserverConfigBase, Prototype.F
     @Option.DefaultCode("new @java.util.ArrayList@(@java.util.List@.of(PathTracingConfig.builder()\n"
             + "                                  .path(\"/metrics/*\")\n"
             + "                                  .tracingConfig(TracingConfig.DISABLED)\n"
-            + "                                  .build(), "
+            + "                                  .build(), \n"
+            + "                                  PathTracingConfig.builder()\n"
+            + "                                  .path(\"/observe/metrics/*\")\n"
+            + "                                  .tracingConfig(TracingConfig.DISABLED)\n"
+            + "                                  .build(), \n"
             + "                                  PathTracingConfig.builder()\n"
             + "                                  .path(\"/health/*\")\n"
             + "                                  .tracingConfig(TracingConfig.DISABLED)\n"
-            + "                                  .build(),"
+            + "                                  .build(), \n"
+            + "                                  PathTracingConfig.builder()\n"
+            + "                                  .path(\"/observe/health/*\")\n"
+            + "                                  .tracingConfig(TracingConfig.DISABLED)\n"
+            + "                                  .build(), \n"
             + "                                  PathTracingConfig.builder()\n"
             + "                                  .path(\"/openapi/*\")\n"
             + "                                  .tracingConfig(TracingConfig.DISABLED)\n"
+            + "                                  .build(), \n"
+            + "                                  PathTracingConfig.builder()\n"
+            + "                                  .path(\"/observe/openapi/*\")\n"
+            + "                                  .tracingConfig(TracingConfig.DISABLED)\n"
             + "                                  .build()))")
-    List<PathTracingConfig> pathConfigs();
+    List<PathTracingConfig> paths();
```

Comment:
This is a backward incompatible change in public API.
We can either consider it a bugfix (and alignment to docs), and change it, or we could have both methods (deprecate the `pathConfigs` and add a builder observer to correctly choose the right configured option)

Reply:
Yes, I considered it a bug fix but I guess in my mind the real bug is the config key being `path-configs` instead of `paths`.

Another option: we could keep the old method name `pathConfigs` but set the config key explicitly to `paths` in the annotation.

Reply:
I think we need to keep the API backward compatible, so please either add a new method and deprecate existing (preferred option), or change the configuration key for this method.

Reply:
The most recent push sets the config key explicitly to "paths" and changes the method names back to what they were to avoid the backward-incompatible change to the APIs.